test/api.js

```diff
@@ -4,8 +4,11 @@ var describe = require('mocha').describe
 
 var posthtml = require('../lib')
 
-var walk = require('../lib/api').walk
-var match = require('../lib/api').match
+var Api = require('../lib/api')
```

Comment:
`API`

Reply:
no, this is constructir notation. this not constant

Reply:
hmm... yeah it's an habit of mine to uppercase letters for a ctor, if `length =< 3`. Both a correct